dev-peerdb.sh

```diff
@@ -24,4 +24,4 @@ if test -n "$USE_PODMAN"; then
 fi
 
 export PEERDB_VERSION_SHA_SHORT=local-$(git rev-parse --short HEAD)
-exec $DOCKER compose $EXTRA_ARGS -f docker-compose-dev.yml up --build
+exec $DOCKER compose -f docker-compose-dev.yml up --build $EXTRA_ARGS
```

Comment:
rename EXTRA_ARGS for podman to PODMAN_RUN_ARGS, have podman set `EXTRA_ARGS=""`